tasks/7_random-fraction/random-fraction.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * randomFraction - generates random decimal number in range 0 to 1
+ *
+ * @return {Number}
+ */
+export function randomFraction() {
+  return Math.random();
```

Comment:
Math.random return some float value, for example 0.16829963774021406
By the task function should return decimal value, for example 0.2